### PR TITLE
8.0 ddt shipping costs

### PR DESCRIPTION
**Pull request by scigghia**
check for company grid with ddt amount ranges, then add related product before punt in pack action.
so we got automatically shipment costs

it is necessary this pr https://github.com/OCA/stock-logistics-workflow/pull/301  to install the module

### CODE REVIEW

File: l10n_it_ddt_shipping_costs/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Andrea Cometa - Apulia Software
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import models
+#from . import wizard
```
Comment (OpenCode) [inline l10n_it_ddt_shipping_costs/__init__.py]: Remove commented line if it's useless

File: l10n_it_ddt_shipping_costs/models/stock_picking_package_preparation.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# Copyright 2017 Andrea Cometa - Apulia Software
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class StockPickingPackagePreparation(models.Model):
+
+    _inherit = 'stock.picking.package.preparation'
+    _rec_name = 'display_name'
```
Comment (SimoRubi) [inline l10n_it_ddt_shipping_costs/models/stock_picking_package_preparation.py]: Questo non credo sia necessario e rischi di sovrascrivere il `_rec_name` definito nei parent.

File: l10n_it_ddt_shipping_costs/views/partner_view.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+    <record model="ir.ui.view" id="partner_shipping_costs_view">
+        <field name="name">res.partner.shipping.costs</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
```
Comment (SimoRubi) [inline l10n_it_ddt_shipping_costs/views/partner_view.xml]: Dovresti ereditare da `l10n_it_ddt.view_partner_form_ddt`

**Comment by eLBati** [comment]
Chiudo per inattività

**Pull request closed without merging**